gravitee-apim-cypress/cypress/commands/management/api-management-commands.ts

```diff
@@ -158,3 +159,24 @@ export function updateApi(auth: BasicAuthentication, apiId: string, apiUpdate: U
     failOnStatusCode,
   });
 }
+
+// export function updateApiSubscription(auth: BasicAuthentication, apiId: string, subscriptionId: string, subscription: Partial<ProcessSubscriptionEntity>) {
+export function updateApiSubscription(auth: BasicAuthentication, apiId: string, subscriptionId: string, subscription: {}) {
+  return cy.request({
+    method: 'POST',
+    url: `${Cypress.config().baseUrl}${Cypress.env('managementApi')}/apis/${apiId}/subscriptions/${subscriptionId}/_process`,
+    auth,
+    headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+    body: `${JSON.stringify(subscription)}`,
```

Comment:
I wonder if it's useful to manually stringify the object to Json here, and to explicit the content type header.
For example, in the POST request in `createPlan`, we don't need that.

Reply:
Yes, that's true, I just faced some problems in this case. So, I was trying to enforce certain settings here but in the end the function was not needed anymore but I left it there for future use. However, I took your remarks into account and changed it accordingly.